README.md

```diff
@@ -1,3 +1,5 @@
+<p align="center"><img height="300px" src="./logo-text.svg" alt="logo"></p>
+
 # Block Editor
 
 This package is a Work In Progress. It aims to seperate the Javascript frontend from [Laraberg](https://github.com/VanOns/laraberg) so it can be maintained seperately, and maybe serve as a starting point for other backend implementations.
```

Comment:
* Do we still need to define this package as a WIP?
* Can this text possibly be modernized?